agent/functional_tests/testdata/taskdefinitions/awslogs-secret-support-multiline/task-definition.json

```diff
@@ -0,0 +1,26 @@
+{
+  "family": "test-awslogs-secret-multiline",
```

Comment:
curious, what does this multiline mean in this test exactly?

Reply:
To test if the secret is supported in a log driver generally, e.g. awslogs. In this case, secret will only be supported in optional awslogs log driver options, and only awslogs-multiline-pattern and awslogs-datetime-format options meet this requirement for both EC2 and Fargate launch type. 
For awslogs-multiline-pattern:
"This option defines a multiline start pattern using a regular expression. A log message consists of a line that matches the pattern and any following lines that don’t match the pattern. Thus the matched line is the delimiter between log messages."
Check out this link to see more details:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html
Hope it helps :)